operations/mimir/autoscaling.libsonnet

```
@@ -389,8 +389,16 @@
     weight=1,
     scale_down_period=null,
     extra_triggers=[],
+    container_name='',
+    pod_regex='',
   ):: self.newScaledObject(
     name, $._config.namespace, {
+      local queryParameters = {
+        container: if container_name != '' then container_name else name,
+        namespace: $._config.namespace,
+        pod_matcher: if pod_regex == '' then '' else ',pod=~"%s"' % pod_regex,
```

I would generalise it in `extra_matchers`. It's a mure renaming thing.

```suggestion
        extra_matchers: if pod_regex == '' then '' else ',pod=~"%s"' % pod_regex,
```

---

sounds good, will change it

---

Done in https://github.com/grafana/mimir/pull/8457/commits/df1fc6009c854493bb9e23b1243ae36a296342bb.